paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlDebeziumJsonEventParser.java

```diff
@@ -113,7 +113,7 @@ public MySqlDebeziumJsonEventParser(
                 caseSensitive,
                 computedColumns,
                 new TableNameConverter(caseSensitive),
-                ddl -> Optional.empty(),
+                new MySqlTableSchemaBuilder(new HashMap<>(), caseSensitive, typeMapping),
```

Comment:
Here return empty is because this constructor is used by sync table action, and sync table action won't parse new tables.

Reply:
It's because I integrated the schema builder in change and new table, so the sync table action also requires the schema builder.

https://github.com/apache/incubator-paimon/blob/7b67e550d548c4297e05fee9e65795e849135f9f/paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlDebeziumJsonEventParser.java#L208-L209